.github/pull_request_template.md

```diff
@@ -93,6 +93,9 @@ Include screenshots if applicable to help explain these changes.
   - [ ] The change requires migration steps
   - [ ] The change upgrades CRDs
   - [ ] The change updates the config *and* the schema
+- Documentation checks:
+  - [ ] The [public documentation](https://github.com/elastisys/compliantkubernetes) required no updates
+  - [ ] The [public documentation](https://github.com/elastisys/compliantkubernetes) required an update - [link to change](set-me\)
```

Comment:
I guess the `\` after `set-me` is a typo.

Reply:
No it is actually intended to force it to render as [link to change](set-me\) instead of [link to change](set-me) so it looks more obvious that it should be replaced. :wink:

Reply:
Smart!